transforms/__testfixtures__/class/class-initial-state.output.js

```diff
@@ -223,7 +225,7 @@ class DeferStateInitialization extends React.Component {
 
 var helper = () => {};
 
-var PassGetInitialState = React.createClass({ // bail out here
+var PassGetInitialState = ReactCreateClass({ // bail out here
```

Comment:
Wrote it this way because it's like a constructor, but don't care either way. Whatever convention we use should match https://github.com/babel/babel/pull/5554